lib/utils/renameService.test.js

```diff
@@ -0,0 +1,108 @@
+'use strict';
+
+const expect = require('chai').expect;
+const Serverless = require('../Serverless');
+const testUtils = require('../../tests/utils');
+const fse = require('fs-extra');
+const path = require('path');
+
+const { renameService } = require('./renameService');
+
+describe('renameService', () => {
+  let serverless;
+  let cwd;
+
+  let servicePath;
+  let newServicePath;
+
+  beforeEach(() => {
+    const tmpDir = testUtils.getTmpDirPath();
+    cwd = process.cwd();
+
+    fse.mkdirsSync(tmpDir);
+    process.chdir(tmpDir);
+
+    servicePath = tmpDir;
+    newServicePath = path.join(servicePath, 'new-service-name');
+
+    serverless = new Serverless();
+    serverless.init();
+  });
+
+  afterEach(() => {
+    // change back to the old cwd
+    process.chdir(cwd);
+  });
+
+  it('should set new service in serverless.yml and name in package.json', () => {
+    const defaultServiceYml =
+      'service: service-name\n\nprovider:\n  name: aws\n';
+    const newServiceYml =
+      'service: new-service-name\n\nprovider:\n  name: aws\n';
+
+    const defaultServiceName = 'service-name';
+    const newServiceName = 'new-service-name';
+
+    const packageFile = path.join(servicePath, 'package.json');
+    const serviceFile = path.join(servicePath, 'serverless.yml');
+
+    serverless.utils.writeFileSync(packageFile, { name: defaultServiceName });
+    fse.writeFileSync(serviceFile, defaultServiceYml);
```

Comment:
I think you might need to remove those files after tests

Reply:
I didnt see other tests removing the temp directory. Do you have an example of another test?

Reply:
Woudn't it be even better to stub the file access methods in the implementation under testing? Having unit tests that rely on any physical filesystem seems odd (we do not want to test `fs` or `fs-extra` - we should assume these work).
PS: I know that this is also the case in other areas of the unit tests, but that does not mean to continue with that habit 😄